index.html

```diff
@@ -1,22 +1,33 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
     <!-- Title -->
-    <link rel = "icon" 
-        href = "./assets/images/leaf.png" 
-        type = "image/x-icon">
+    <link rel="icon" href="./assets/images/leaf.png" type="image/x-icon" />
     <title>Todos</title>
 
     <!-- css -->
-    <link rel="stylesheet" href="./assets/css/style.css">
-</head>
-<body>
-    
+    <link rel="stylesheet" href="./assets/css/style.css" />
+  </head>
+  <body>
+    <h1>Add Tasks</h1>
+
+    <button id="create">+ Create</button>
```

Comment:
Can we use `fontawesome icon` instead o hard coded plus+ here? See [how-to](fontawesome)